build_badtrack.py

```diff
@@ -83,6 +83,8 @@
 os.chmod(f"{APP_PATH}/badtrack/DEBIAN/postinst", 0o755)
 
 # Build the Debian package
-subprocess.run(["dpkg-deb", "--build", f"{APP_PATH}/badtrack"])
-
-print("badtrack.deb package has been created.")
+try:
+    subprocess.run(["dpkg-deb", "--build", f"{APP_PATH}/badtrack"],check=True)
+    print("badtrack.deb package has been created.")
+except subprocess.CalledProcessError:
+    print("package could not be built.")
```

Comment:
No need to catch, let the exception happen so we know something went wrong. Or if you really want to print "package could not be build", then at least re-raise the exception by just doing `raise`:
```python
try:
    subprocess.run(["dpkg-deb", "--build", f"{APP_PATH}/badtrack"],check=True)
    print("badtrack.deb package has been created.")
except subprocess.CalledProcessError:
    print("package could not be built.")
    raise
```

Reply:
ahh I just got confused by the previous comment because I already added check on my local version so I thought you didn't want the exception